var/spack/repos/builtin/packages/oce/package.py

```diff
@@ -44,6 +44,8 @@ class Oce(Package):
 
     variant('tbb', default=True,
             description='Build with Intel Threading Building Blocks')
+    variant('X11', default=False,
+            description='Build with X11 enabled')
```

Comment:
these two PR might be of relevance: https://github.com/LLNL/spack/pull/4258 https://github.com/LLNL/spack/pull/4134